### PR TITLE
feat: :lipstick: release redesigned landing page

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [ ] PR closes #<issue_number>
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-10-19 at 11 26 24

### CODE REVIEW

**Comment by preschian** [comment]
this is the PR when we ready to release the new design to the public without params
cc @yangwao

**Review by roiLeo — COMMENTED** [review]
`KodadotSocialLinks` isn't used? :( not even in footer?
You can remove/rename `TopCollection` component (new is `ReTopCollection`)

**Comment by preschian** [comment]
> `KodadotSocialLinks` isn't used? :( not even in footer? 

it seems no component is using it. in the new footer use the array list https://github.com/kodadot/nft-gallery/blob/main/components/TheFooter.vue#L126

**Comment by yangwao** [comment]
> this is the PR when we ready to release the new design to the public without params

oh, perfect, been looking for this one too :)

I guess we are close to wrapping up once there will be navbar 
- #4063

File: components/Navbar.vue [hunk 1]
```diff
@@ -254,10 +254,6 @@ export default class NavbarMenu extends mixins(
     return this.$store.getters['history/getCurrentlyViewedItem']?.name || ''
   }
 
-  get isRedesignedLandingPage() {
-    return this.$route.name === 'index' && this.redesign
-  }
```
Comment (Jarsen136) [inline components/Navbar.vue:259]: Hey, pls look carefully. The condition is not only `isRedesign`.
So, you could not remove this function and also the search bar component.

```ts
get isLandingPage() {
     return this.$route.name === 'index'
}
```

Comment (preschian) [reply components/Navbar.vue:259]: oops, I missed this one

**Comment by vikiival** [comment]
Setting up as draft until #4063 is merged

**Comment by yangwao** [comment]
let's resolve conflicts @preschian and let it fly

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
On focus in searchbox we should suggest trending collections 

image

Will make extra issue

**Review by yangwao — APPROVED** [review]
finally!

**Pull request merged**